models/user/src/mc-user/types.ts

```diff
@@ -0,0 +1,69 @@
+import type { TBuilder } from '@commercetools-test-data/core';
+import type { TMcProject } from '@commercetools-test-data/project';
+
+type ProjectQueryResult = {
+  count: number;
+  offset: number;
+  total: number;
+  results: TMcProject[];
+};
+
+export type TIdTokenUserInfo = {
+  iss: string;
+  sub: string;
+  aud: string;
+  exp: number;
+  iat?: number;
+  email?: string;
+  name?: string;
+  additionalClaims?: string;
+};
+
+export type TIdTokenUserInfoGraphql = TIdTokenUserInfo & {
+  __typename: 'IdTokenUserInfo';
+};
+
+export type TMcUser = {
```

Comment:
Nit: would be nice to use the graphql generated types (codegen). Can be also a follow up

Reply:
Yep, generated both core and mc [acf9a71](https://github.com/commercetools/test-data/pull/466/commits/acf9a715fb4684c21030b3916fe0cfbbe2af714f), [32e29bc](https://github.com/commercetools/test-data/pull/466/commits/32e29bc0354cb2c9fe6144c2f029a3dd45f5b6cb)